sass/reset.scss

```diff
@@ -1,4 +1,10 @@
-/* reset & clearfix */
+/* reset & clearfix
```

Comment:
Per my comments here, I've removed the outline styles from the reset. This is something that I would want to run by a designer to see if its suitable for the site. It's important to keep in mind each browser will implement the focus style differently but this may be for the best. Users will be most familiar with their browsers focus style.